docs/bookmarks.md

```diff
@@ -0,0 +1,221 @@
+# Bookmarks
+
+
+## Introduction
+
+Bookmarks are named pointers to revisions (just like branches are in Git). You 
+can move them without affecting the target revision's identity. Bookmarks
+automatically move when revisions are rewritten (e.g. by `jj rebase`). You can
+pass a bookmark's name to commands that want a revision as argument. For example,
+`jj new main` will create a new revision on top of the "main" bookmark. Use
+`jj bookmark list` to list bookmarks and `jj bookmark` to create, move, or delete
+bookmarks. There is currently no concept of an active/current/checked-out bookmark.
+
+Currently Jujutsu maps its Bookmarks to Git Branches and stores them as that 
```

Comment:
Not that this is important, but I think this is not quite true AFAIK. In a normal repo, I think only remote-tracking branches are stored in the Git backend, unless you explicitly run `jj git import/export`.

One thing we could say here:

> In a Jujutsu repo that's colocated with git, Jujutsu's bookmarks map to Git branches.

Reply:
If we decide to keep talking about "remote bookmarks" (the opposite of my proposed edits), we'd probably need more explanation somewhere around here of the fact that remote bookmarks are remote branches for Git remotes. If we switch back to "remote branches", I think there's no need (though other places might need some editing).

Reply:
Resolving all your comments to do it in a follow-up.